share/spice/flash_version/flash_version.js

```diff
@@ -1,34 +1,35 @@
 (function(env) {
     "use strict";
-
+    
     env.ddg_spice_flash_version = function() {
-
-	if(!FlashDetect) {
-	    return Spice.failed('flash_version');
-	}
-
-	// Display the plugin.
-	Spice.add({
-            data: {
-		installed: FlashDetect.installed,
-		raw: FlashDetect.raw
-	    },
-	    id: 'flash_version',
-	    name: 'Software',
-	    meta: {
-		sourceName: 'Adobe',
-		sourceUrl: 'https://get.adobe.com/flashplayer/',
-		sourceIcon: true
-	    },
-            templates: {
-		group: 'base',
-		options: {
-		    content: Spice.flash_version.content,
-		    moreAt: true
-		}
+        
+        DDG.require('flash_detect.js', function(){
```

Comment:
I ended up going with `flashDetect` as the name, can you please change this to `DDG.require('flashDetect')`